server.js

```diff
@@ -1,24 +1,102 @@
-const app = require('express')();
```

Comment:
I like this one better but that's just my preferential thing

Reply:
I needed access to the express library to import `use(express.json())` a few lines down, unfortunately. I also loved the style way more!